MAINTAINERS.md

```diff
@@ -15,6 +15,10 @@ Furthermore, for changes affecting one or more implementations, ensure implement
 * [Chrome](https://crbug.com/new)
 * [Firefox](https://bugzilla.mozilla.org/enter_bug.cgi?product=Core&component=DOM)
 * [Safari](https://bugs.webkit.org/enter_bug.cgi?product=WebKit&component=HTML%20DOM)
+* [Deno](https://github.com/denoland/deno/issues/new) ^1
```

Comment:
It seems that per https://docs.github.com/en/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#footnotes you need square brackets as well for this?

Reply:
Didn't know GH supported this. Cool!